### PR TITLE
Create cpp remainder entry, format all and lint

### LINKED ISSUE #984: [Term Entry] C++ Math Functions: `remainder()`
### Reason/inspiration (optional)

Under `cpp/concepts/math-functions/terms/`, create a new term entry for the `remainder()` function.

- In the terms/ folder, make a new folder called `remainder`
- Inside the `remainder` folder, create a new markdown file called `remainder.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Let's include an "Example" section with a \```cpp code snippet.
- Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [X] Issue is assigned

### PR DESCRIPTION
**Pull request by ddiegow**
### Description

Add cpp remainder() entry

Closes #984

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
Already approved the prior version of this.

**Comment by ddiegow** [comment]
@SSwiniarski  I'm quite new to github workflows in general, do I need to do anything else after this?

**Comment by SSwiniarski** [comment]
@ddiegow There will be a second review by one of the Maintainers. Once that's resolved we should be ready to merge.

File: content/cpp/concepts/math-functions/terms/remainder/remainder.md
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/remainder/remainder.md:66]:
```suggestion
#include <iostream>
#include <cmath>
using namespace std;

int main () {
  cout << "Remainder of 7.3 / 2 is " << remainder(7.3, 2) << endl;
  return 0;
}
```

**Comment by ddiegow** [comment]
Thanks for the suggestions @Dusch4593. I'll work on them this weekend :)

File: content/cpp/concepts/math-functions/terms/remainder/remainder.md
Comment (ddiegow) [reply content/cpp/concepts/math-functions/terms/remainder/remainder.md:50]: Added the explanation.

**Review by Dusch4593 — APPROVED** [review]
@ddiegow LGTM! 👍🏻

Made a few small changes in 1142029146c29e2c7ab290b99f194b73467cbaf0, ad6905217b61205167e9339592e36962b64e05f9, and dfcf992051edee4568b0a954012293c12f7defce:

- Edited lines # 3 and # 15 to more closely resemble each other.
- Inline references to functions and parameters were enclosed in backticks.
- Added some colons `:` under the example sections.

cc: @SSwiniarski

**Comment by Dusch4593** [comment]
Congrats and thank you so much for contributing to Codecademy Docs, @ddiegow! 🎊 🎉 🥳

The entry should now be live: https://www.codecademy.com/resources/docs/cpp/math-functions/remainder

Currently, your Codecademy profile is listed anonymously on the entry page. To make it visible, you'll need to add your GitHub username to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account:

Screen Shot 2022-12-03 at 9 06 27 AM

Looking forward to your next Docs contribution! 😄

**Pull request merged**
